user/src/pages/index.tsx

```diff
@@ -1,5 +1,40 @@
-import { Typography } from "@material-ui/core";
+import "fontsource-roboto";
```

Comment:
The font family for the project was already defined in `_app.tsx`. Remove this import statement.